### PR TITLE
Remove Safari 6.1

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes Safari 6.1 from the data and converts all of its entries to Safari 7.  Details are described in https://github.com/mdn/browser-compat-data/issues/9423, but to summarize, it seems that Safari 6.1 and 6.2 are simply backports of Safari 7.0 and 7.1 for older macOS versions, similar to [Safari 4.1 and 5.0](https://github.com/mdn/browser-compat-data/issues/4679).

### CODE REVIEW

**Comment by foolip** [comment]
I looked into making this changes yesterday on a branch, which I've created as https://github.com/mdn/browser-compat-data/pull/11157 now. That has a bunch of different commits, and only "Change where iOS is 7" is trivial IMHO. I'd especially want to look into the cases where iOS said 6.1, since that has the same WebKit version as iOS 6. I don't really believe that data, but if we did we'd change iOS 6.1 to 6, not 7. So for those we need to research what it should really be.

File: api/Window.json [hunk 1]
```diff
@@ -1670,10 +1670,10 @@
               "version_added": "24"
             },
             "safari": {
-              "version_added": "6.1"
+              "version_added": "7"
```
Comment (foolip) [inline api/Window.json:1673]: Using http://mdn-bcd-collector.appspot.com/tests/api/Window/crypto it looks like `window.crypto` was actually added in Safari 5.1. Maybe the collector can be used to update a bunch of this data?

**Comment by vinyldarkscratch** [comment]
I'm marking this as "not ready" as I'm planning to talk to Daniel about what we want to do with Safari 6.1.

**Comment by vinyldarkscratch** [comment]
Removing the "not ready" flag -- we've just about come to a conclusion to remove Safari 6.1!

**Comment by foolip** [comment]
@vinyldarkscratch was most of this produced by a script so that you can easily recreate it? If so, could you separate it into multiple commits and PRs to make it easier to review? In particular, all the cases where 6.1+6.1 are changed to 7+7 are ones I don't feel need any review at all. Splitting just that out would probably make the rest small enough to review together.

**Comment by vinyldarkscratch** [comment]
It was generated by a quick `grep` and `sed` replacement, but I could instead run the collector on these and get accurate results?  I can split that into separate PRs for easier review too.

**Comment by foolip** [comment]
A separate PR any APIs that the collector determines were supported *before* 6.1 would be great.

**Comment by vinyldarkscratch** [comment]
I've run a traverse through BCD to see what's currently set to Safari 6.1.  I'll use this PR as a breakdown of what's left to correct (and set it to "not ready" in the meantime):

- [ ] api.Crypto
  - getRandomValues
- [ ] api.CustomEvent
  - CustomEvent
  - detail
- [ ] api.DOMTokenList
  - toggle.force_parameter
- [ ] api.Document
  - Document
  - currentScript
  - origin
  - wheel_event
- [ ] api.Element
  - mouseenter_event
  - mouseleave_event
- [ ] api.FileReader
  - error
  - onabort
  - onerror
  - onload
  - onloadend
  - onprogress
  - readyState
- [ ] api.HTMLImageElement
  - srcset
  - height
  - width
  - audioTracks
  - fastSeek
  - videoTracks
- [ ] api.HTMLTemplateElement
  - content
- [ ] api.Range
  - Range
- [ ] api.TextTrackCue
  - endTime
  - id
  - pauseOnExit
  - startTime
  - track
- [ ] api.URL
  - toString
- [ ] api.VTTCue
  - VTTCue
  - align
  - getCueAsHTML
  - line
  - position
  - size
  - snapToLines
  - text
  - vertical
- [ ] api.WheelEvent
  - WheelEvent
  - deltaMode
- [ ] api.Window
  - cancelAnimationFrame
  - crypto
  - requestAnimationFrame
  - requestAnimationFrame.return_value

**Comment by queengooborg** [comment]
Current remaining entries are:

→ css.properties.align-self.flex_context
→ css.properties.clip-path
→ css.properties.clip-path.basic_shape
→ css.properties.clip-path.html
→ css.properties.clip-path.svg
→ css.properties.display.inline-flex
→ css.properties.flex-wrap
→ css.properties.font-feature-settings
→ css.properties.justify-content.flex_context
→ css.properties.min-height.fit-content
→ css.properties.position.sticky
→ css.properties.ruby-position
→ css.properties.width.animatable
→ css.properties.width.stretch
→ css.properties.word-spacing.percentages
→ css.selectors.cue
→ css.selectors.cue.selector_argument
→ css.selectors.future
→ css.selectors.past
→ css.types.image.gradient.linear-gradient
→ css.types.image.gradient.linear-gradient.interpolation_hints
→ css.types.image.gradient.linear-gradient.to
→ css.types.image.gradient.linear-gradient.unitless_0_angle
→ css.types.image.gradient.radial-gradient
→ css.types.image.gradient.radial-gradient.interpolation_hints
→ css.types.image.gradient.repeating-linear-gradient
→ css.types.image.gradient.repeating-linear-gradient.interpolation_hints
→ css.types.image.gradient.repeating-linear-gradient.to
→ css.types.image.gradient.repeating-linear-gradient.unitless_0_angle
→ css.types.length-percentage.vmax
→ css.types.length-percentage.vmin
→ css.types.length.vmax
→ css.types.length.vmin
→ html.elements.details
→ html.elements.details.open
→ javascript.builtins.Array.toLocaleString.locales_parameter
→ javascript.builtins.Array.toLocaleString.options_parameter
→ svg.elements.use.external_uri

**Comment by queengooborg** [comment]
The remaining features are all covered by #13867, #13868, #13869, #13870.

**Comment by foolip** [comment]
@queengooborg those PRs are merged, this now need rebasing.

**Comment by queengooborg** [comment]
Oops, one more: #13888!

**Comment by foolip** [comment]
Also merged!

**Comment by queengooborg** [comment]
Alright, this is now ready to go!  Whoo!

**Review by foolip — APPROVED** [review]

**Pull request merged**